android/src/ch/opengis/qfield/QFieldProjectActivity.java

```diff
@@ -140,7 +140,12 @@ private void drawView() {
                 for (File file : list) {
                     if (file.getName().startsWith(".")) {
                         continue;
-                    }else if (file.getName().toLowerCase().endsWith(".qgs") || file.getName().toLowerCase().endsWith(".qgz")){
+                    }else if (file.getName().toLowerCase().endsWith(".qgs") || file.getName().toLowerCase().endsWith(".qgz") ||
+                              file.getName().toLowerCase().endsWith(".gpkg") || file.getName().toLowerCase().endsWith(".shp") || 
+                              file.getName().toLowerCase().endsWith(".kml") || file.getName().toLowerCase().endsWith(".kmz") || 
+                              file.getName().toLowerCase().endsWith(".tif") || file.getName().toLowerCase().endsWith(".jpg") || 
+                              file.getName().toLowerCase().endsWith(".png") || file.getName().toLowerCase().endsWith(".pdf") ||
+                              file.getName().toLowerCase().endsWith(".gpx")){
```

Comment:
```suggestion
                    } else if (
                        file.getName().toLowerCase().endsWith(".qgs")
                        || file.getName().toLowerCase().endsWith(".qgz")
                        || file.getName().toLowerCase().endsWith(".gpkg")
                        || file.getName().toLowerCase().endsWith(".shp")
                        || file.getName().toLowerCase().endsWith(".kml")
                        || file.getName().toLowerCase().endsWith(".kmz")
                        || file.getName().toLowerCase().endsWith(".tif")
                        || file.getName().toLowerCase().endsWith(".jpg")
                        || file.getName().toLowerCase().endsWith(".png")
                        || file.getName().toLowerCase().endsWith(".pdf")
                        || file.getName().toLowerCase().endsWith(".gpx")
                        || file.getName().toLowerCase().endsWith(".geojson")
                    ) {
```
This will reduce the diff noise later, also added geojson to the list.

Reply:
Please check this one here https://github.com/opengisch/QField/pull/1641/files#r567241061